src/main/java/trello/keys/NamePages.java

```diff
@@ -42,7 +41,6 @@ public NamePages(final User user) {
      */
     public String getHomePage() {
```

Comment:
Change the name of this method please. E.G.  getUriHomePage()

Reply:
done.